pkg/tenant/api/v1/client.go

```diff
@@ -518,10 +518,6 @@ func (s *APIv1) ProjectCreate(ctx context.Context, in *Project) (out *Project, e
 }
 
 func (s *APIv1) ProjectDetail(ctx context.Context, id string) (out *Project, err error) {
-	if id == "" {
-		return nil, ErrProjectIDRequired
-	}
```

Comment:
Unless I'm mistaken; If `id == ""` then the request will be made to `GET /v1/projects/` based on the `fmt.Sprintf` on line 521; the response will likely return a 200; but it won't contain a `*Project` but a `*ProjectPage` -- do we need to handle this error condition?

Reply:
Thanks, this was leftover from a previous version of the PR.